tests/plugins/widget/manual/deleteparagraph.html

```diff
@@ -0,0 +1,50 @@
+<div id="editor">
+	<div class="simplebox">
+		<h2 class="simplebox-title">Title</h2>
+
+		<div class="simplebox-content">
+			<p>Content...</p>
+		</div>
+	</div>
+	<p>&nbsp;</p>
+    <pre><code class="language-javascript">//Hello another world!</code></pre>
+</div>
+<script>
+	if ( bender.tools.env.mobile ) {
+		bender.ignore();
+	}
+
+	CKEDITOR.plugins.add( 'simplebox', {
+		requires: 'widget',
+		icons: 'simplebox',
+		init: function( editor ) {
+			editor.widgets.add( 'simplebox', {
+				allowedContent: 'div(!simplebox); div(!simplebox-content); h2(!simplebox-title)',
+				requiredContent: 'div(simplebox)',
+				editables: {
+					title: {
+						selector: '.simplebox-title',
+						allowedContent: 'br strong em'
+					},
+					content: {
+						selector: '.simplebox-content',
+						allowedContent: 'p br ul ol li strong em'
+					}
+				},
+				template:
+				'<div class="simplebox">' +
+				'<h2 class="simplebox-title">Title</h2>' +
+				'<div class="simplebox-content"><p>Content...</p></div>' +
+				'</div>',
+				button: 'Create a simple box',
+				upcast: function( element ) {
+					return element.name == 'div' && element.hasClass( 'simplebox' );
+				}
+			} );
+		}
+	} );
+	var editor = CKEDITOR.replace('editor', {
+		extraPlugins: 'simplebox',
+		extraAllowedContent: 'pre'
```

Comment:
With `codesnippet` plugin loaded, we don't need to add `pre` here - it is handled by the plugin itself.